spark/common/src/main/scala/org/apache/sedona/sql/utils/RasterSerializer.scala

```diff
@@ -42,4 +43,18 @@ object RasterSerializer {
   def deserialize(value: Array[Byte]): GridCoverage2D = {
     Serde.deserialize(value);
   }
+
+  def serializeSampleDimension(sampleDimension: GridSampleDimension): Array[Byte] = {
+    val bos = new ByteArrayOutputStream()
+    val oos = new ObjectOutputStream(bos)
+    oos.writeObject(sampleDimension)
+    oos.flush()
+    bos.toByteArray
+  }
+
+  def deserializeSampleDimension(bytes: Array[Byte]): GridSampleDimension = {
+    val bis = new ByteArrayInputStream(bytes)
+    val ois = new ObjectInputStream(bis)
+    ois.readObject().asInstanceOf[GridSampleDimension]
+  }
```

Comment:
We have a `GridSampleDimensionSerializer` implemented using Kryo. I recommend using that instead of the Java serializer.